src/style.css

```diff
@@ -0,0 +1,26 @@
+.body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
There shouldn't be margin right after last star in the row

Reply:
Done